.jenkins/lm-eval-harness/configs/Llama-2-7B-hf.yaml

```diff
@@ -0,0 +1,14 @@
+# These scores were chosen to place within 6% range of values achieved using  vLLM on HPU:
+# 0.148 - 0.164
+# where on https://www.llama.com/llama2/: 0.146 is given
+model_name: "/mnt/weka/data/pytorch/llama2/Llama-2-7b-hf"
```

Comment:
Do we really want to you this model? IMO we should use "/mnt/weka/data/pytorch/llama2/Llama-2-7b-chat-hf"

Reply:
We still have to support and improve LLama2 (as far as I know). I will change the model. Thanks

Reply:
I received bigger than +- 6% spread of results for chat model, so I decided to revert to the original one.